5EUNG/app.js

```diff
@@ -0,0 +1,54 @@
+//// 3rd-party package
+require('dotenv').config();
+const express = require('express');
+const cors = require('cors');
+const morgan = require('morgan');
+
```

Comment:
dotenv 객체 호출 이후 줄바꿈을 해주세요!
```suggestion
require('dotenv').config();

const express = require('express');
const cors = require('cors');
const morgan = require('morgan');

```